vibe_code_apps_19/app/agent_api.py

```
@@ -754,8 +763,10 @@ def export_agent_bundle(
             path = out / "data_model.csv"
             dm.to_csv(path, index=False)
             written["data_model"] = path
-    except Exception:
-        pass
+    except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)